src/java.base/share/classes/java/util/Locale.java

```diff
@@ -1210,6 +1211,23 @@ public static Locale[] getAvailableLocales() {
         return LocaleServiceProviderPool.getAllAvailableLocales();
     }
 
+    /**
+     * {@return A stream of installed locales.}
+     *
+     * The returned stream represents the union of locales supported
+     * by the Java runtime environment and by installed
+     * {@link java.util.spi.LocaleServiceProvider LocaleServiceProvider}
+     * implementations. At a minimum, the returned stream must contain a
+     * {@code Locale} instance equal to {@link Locale#ROOT Locale.ROOT} and
+     * a {@code Locale} instance equal to {@link Locale#US Locale.US}.
+     * @implNote Unlike {@code getAvailableLocales()}, this method does
```

Comment:
I'd insert a blank line between the method description paragraphs and the following javadoc tags.